docs/roadmap.md

```diff
@@ -0,0 +1,36 @@
+# Long-term Roadmap
+
+This is the long-term, draft Roadmap. Note that this a vision document that reflects our
+current desires. It is not a commitment to implement everything listed in this roadmap.
+The primary purpose of this document is to ensure all contributors work in alignment.
+As our vision changes over time maintainers reserve the right to modify this roadmap,
+add and _remove_ items from it.
+
+Description|Status|Links|
+-----------|------|-----|
+**Testing**|
+Add metric testing to the testbed|Done|
```

Comment:
nit: I'm not sure about the value of keeping the done items here, anyway, we can remove the old ones as they get stale.

Reply:
I think it is useful to keep recently done items in this list. It helps to show the direction of the development: where we are coming from. We can retire items from this document as they become older than a couple months.